docs/cip/README.md

```diff
@@ -4,704 +4,2873 @@ Title: Ouroboros Leios - Greater transaction throughput
+
+### How Leios addresses CPS-18
+
+The [Leios research paper][leios-paper] describes a highly concurrent protocol
+with three block types - Input Blocks (IBs), Endorser Blocks (EBs), and Ranking
+Blocks (RBs) - produced independently across decoupled, pipelined stages. This
+specification simplifies that design by eliminating IBs and coupling EB
+production with RB production, reducing complexity while preserving substantial
+throughput gains.
+
+This simplification avoids the complexity and ecosystem disruption of
+implementing massive throughput increases immediately, while still delivering
+substantial gains to address [CPS-18 Greater Transaction Throughput][cps-18]
+challenges. Four strategic design priorities guided this approach:
+
+1. [Economic sustainability](#economic-sustainability)
+2. [Reasonable time to market](#time-to-market)
+3. [Minimal downstream impact](#downstream-impact)
+4. [Competitive positioning](#competitiveness)
+
+<a name="economic-sustainability"></a>**1. Economic sustainability: Capacity
+without utilization risk**
+
+On one hand, this approach avoids over-engineering massive throughput capacity
+without proven demand. Creating fundamental system changes to support multiple
+orders of magnitude more throughput adds to the cost of running a more
+expensive, more capable system that does not pay for itself until utilization
+increases.
+
+On the other hand, the minimum economic requirement establishes the lower bound.
+As the Cardano Reserve diminishes, transaction fees must replace rewards to
+maintain network security and SPO profitability. Currently, the Reserve
+contributes more than 85% of epoch rewards, with less than 15% coming from
+transaction fees. By 2029, to compensate for Reserve depletion, the network
+requires approximately 36-50 TPS with average-sized transactions - roughly 10
+times current mainnet throughput. This conservative lower bound represents the
+breakeven point for running the protocol sustainably.
+
+However, TPS is not an appropriate metric for defining these bounds. To properly
+assess economic breakeven points, we measure throughput in Transaction Bytes per
+second (TxB/s) rather than Transactions per second (TPS). TPS does not account
+for transaction size or computational complexity, making systems with smaller
+transactions appear "faster" while providing less utility. Current Cardano
+mainnet provides 4,500 TxB/s, while this specification targets 140,000-300,000
+TxB/s (equivalent to roughly 100-200 TPS) - a 30-65x increase sufficient for
+economic sustainability.
+
+<div align="center">
+<a name="figure-8" id="figure-8"></a>
+<p>
+  <img src="images/leios-forecast-sqrt-fill.svg" alt="SPO profitability forecast under Leios">
+</p>
+
+<em>Figure 8: SPO profitability forecast under Leios showing clear economic
+benefits once sustained throughput exceeds 50-70 TxkB/s (36-50 TPS
+equivalent)</em>
+
+</div>
+
+Achieving this capacity increase requires trade-offs, as detailed below.
```

Comment:
This sentence is very lost and does not add much value alone. Would it make sense to make it part of the paragraph above the figure or just drop it?